ir/function.cpp

```diff
@@ -48,6 +49,41 @@ JumpInstr::it_helper BasicBlock::targets() const {
   return {};
 }
 
+void BasicBlock::prunePhiSrc(const BasicBlock *bb, bool isInclusive) {
+  for (auto &i : instrs()) {
+    Phi *phi = dynamic_cast<Phi*>(const_cast<Instr*>(&i));
+    if (!phi) break;
```

Comment:
bb may block with non-phi but have phis. Try this:
```
@a = global i32 0

define i8* @src(i1 %cond) {
  br i1 %cond, label %A, label %B
A:
  br label %C
B:
  br label %C
C:
  %v = phi i8* [ bitcast (i32* @a to i8*), %A ], [ null, %B ]
  ret i8* %v
}

```

Reply:
fixed :)